test/widgets/emoji_reaction_test.dart

```diff
@@ -434,7 +434,7 @@ void main() {
       // request the message action sheet
       await tester.longPress(find.byType(MessageContent));
       // sheet appears onscreen; default duration of bottom-sheet enter animation
-      await tester.pump(const Duration(milliseconds: 250));
+      await transitionDurationObserver.pumpPastTransition(tester);
```

Comment:
In the code comment, the part after the semicolon ";" doesn't refer to anything; remove it. (Here and in similar comments on code touched in this PR.)